app.rb

```diff
@@ -0,0 +1,46 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require 'sinatra/flash'
+
+class MakersBnB < Sinatra::Base
+
+  enable :sessions
```

Comment:
enable method override to preempt delete? Understand that may be one for one for later